tests/drivers/gpio/gpio_arduino/README.txt

```diff
@@ -0,0 +1,11 @@
+Title: test_gpio_arduino
```

Comment:
that readme is very barebones and has some custom layout , can you please follow format of existing READMEs coming with tests...

Reply:
Thanks for checking this, I am too lazy here. updated